app/jenkins-for-jira-ui/src/components/JenkinsServerList/ConnectedServer/JenkinsModal.tsx

```diff
@@ -1,26 +1,32 @@
-import Button, { LoadingButton } from '@atlaskit/button';
+import Button, { LoadingButton, Appearance } from '@atlaskit/button';
 import React from 'react';
 import Modal, {
-	Appearance,
 	KeyboardOrMouseEvent,
 	ModalBody,
 	ModalFooter,
 	ModalHeader,
 	ModalTitle,
-	ModalTransition
+	ModalTransition,
+	Appearance as ModalAppearance
 } from '@atlaskit/modal-dialog';
-import { Appearance as ButtonAppearance } from '@atlaskit/button';
+import EditorSuccessIcon from '@atlaskit/icon/glyph/editor/success';
+import { cx } from '@emotion/css';
 import { JenkinsServer } from '../../../../../src/common/types';
```

Comment:
lol lotta ././././././ not now but sometime, might be time we use aliases like GH4J